resources/zh/cemu.po

```diff
@@ -452,76 +555,35 @@ msgstr "设置CPU模拟模式"
 
 #: GameProfileWindow.cpp
 msgid "Thread quantum"
-msgstr "Thread quantum"
+msgstr "Thread quantum | 线程量为"
 
 #: GameProfileWindow.cpp
 msgid ""
 "EXPERT OPTION\n"
 "Set the maximum thread slice runtime (in virtual cycles)"
 msgstr ""
-"专家选项\n"
-"Set the maximum thread slice runtime (in virtual cycles)"
+"专业选项 ⚠\n"
+"设置最大线程量运行（以虚拟周期为单位）"
```

Comment:
I really didn't know much before. 😂This really needs me to think about how to translate it correctly. . . I will check some more information (because the content you posted is understandable, but it is still machine translation), and I will submit it with other changes after consideration. Thank you very much.

Reply:
It might come in handy the original English text that explains thread quantum:
> The QuantumLength property defines the number of clock ticks per quantum. A quantum is a unit of execution time that the scheduler is allowed to give to an application before switching to other applications. When a thread runs one quantum, the kernel preempts it and moves it to the end of a queue for applications with equal priorities. The actual length of a thread's quantum varies across different Windows platforms.